CHANGELOG.md

```diff
@@ -35,6 +35,7 @@
   release. Using it logs a runtime warning.
 - Experimental gnu libc builds use empty default seccomp filters, allowing all
   system calls.
+- Docs: updated [guide](docs/network-setup.md) for network setup
```

Comment:
I don't think we need to mention this in the changelog. Not sure.

Reply:
Indeed, we do Not need to mention this in the changelog.

Reply:
Yeah, I wasn't sure about that either. Removed :)